packages/gatsby-theme-docs/src/templates/release-notes-list.js

```diff
@@ -11,27 +11,68 @@ import markdownFragmentToReact from '../utils/markdown-fragment-to-react';
 import { SEO, ThemeProvider } from '../components';
 import markdownComponents from '../markdown-components';
 
-const ReleaseNotesListTemplate = (props) => (
-  <ThemeProvider>
-    <LayoutReleaseNotesList
-      pageContext={props.pageContext}
-      pageData={props.data.contentPage}
-    >
-      <Markdown.TypographyPage>
-        <SEO
-          title={props.data.contentPage.title}
-          excludeFromSearchIndex={props.data.contentPage.excludeFromSearchIndex}
-        />
-        <MDXProvider components={markdownComponents}>
+const ReleaseNotesListTemplate = (props) => {
+  const [releaseNotes, setReleaseNotes] = React.useState(
+    props.data.allReleaseNotePage ? props.data.allReleaseNotePage.nodes : []
+  );
+  const [fromFilterDate, setFromFilterDate] = React.useState('');
+  const [toFilterDate, setToFilterDate] = React.useState('');
```

Comment:
Shouldn't there be some initial from/to dates? 🤔

Reply:
> Shouldn't there be some initial from/to dates? 🤔

That would mean the list would always be filtered, I don't see any benefit in that implementation.

Reply:
I guess this is more of a general question to @nkuehn : if we have releases going back to 5 years ago, do we want to always show/render all of them?

I would image that we only want to show the more "recent" releases.
Not important for now, but seeing this popped up the question and we can discuss/look into this as a follow up.

Reply:
> Shouldn't there be some initial from/to dates? 🤔

No, there shouldn't be a preset.  It's a different state to have not selected anything vs. having selected today to e.g. send a link to this week's releases that will still work next week. Also, the date of the oldest release is some random date somewhere in the past that does not make sense to users.  

> if we have releases going back to 5 years ago, do we want to always show/render all of them?

Yes, also this question was already discussed and decided in the design process.  We have no problems with the current implementation (which shows even more than we migrate) and chose to not buy into the UI complexity of a paged or "show more" interface.  

If we get into performance issues we can choose to limit the initial list but that needs some UX and design consideration.   It would be a bit of a pity though to see that the performance is regressing.